gremlin-language/src/main/antlr4/Gremlin.g4

```diff
@@ -740,24 +742,28 @@ traversalMethod_write
     ARGUMENT AND TERMINAL RULES
 **********************************************/
 
+structureVertex
+    : NEW ('Vertex'|'ReferenceVertex') LPAREN genericLiteral COMMA stringLiteral RPAREN
```

Comment:
ReferenceVertex can be constructed without a label too in which case the label is automatically assigned to "vertex". See https://github.com/apache/tinkerpop/blob/7f7d3a485c7f100f98047b71672a0c2c9ab855b4/gremlin-core/src/main/java/org/apache/tinkerpop/gremlin/structure/util/reference/ReferenceElement.java#L61 

I think what you are doing is correct, i.e. to have users specify vertex label explicitly but for completeness sake, please add a comment here on why we don't allow a reference vertex with just an ID.

Reply:
added a comment to the grammar file.